src/Contract/Monad.purs

```diff
@@ -0,0 +1,72 @@
+-- | A module defining the `Contract` monad.
```

Comment:
Can we define a `defaultQueryConfig` here (it needs to be effectful since it initializes websockets, in `Aff` not `Contract`)?